sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/orc/OrcScan.scala

```diff
@@ -31,7 +32,8 @@ case class OrcScan(
     hadoopConf: Configuration,
     fileIndex: PartitioningAwareFileIndex,
     dataSchema: StructType,
-    readSchema: StructType) extends FileScan(sparkSession, fileIndex, readSchema) {
+    readSchema: StructType,
+    options: DataSourceOptions) extends FileScan(sparkSession, fileIndex, readSchema, options) {
```

Comment:
If this PR needs to touch `Orc*` classes like this, could you mention in the PR description please?